django/thunderstore/repository/models/namespace.py

```diff
@@ -0,0 +1,42 @@
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.db.models import Manager
+
+from thunderstore.repository.validators import PackageReferenceComponentValidator
+
+
+class Namespace(models.Model):
+    objects: "Manager[Namespace]"
+
+    name = models.CharField(
+        max_length=64,
+        unique=True,
+        validators=[PackageReferenceComponentValidator("Namespace name")],
+    )
+    team = models.ForeignKey(
+        "repository.Team",
+        on_delete=models.CASCADE,
+        related_name="namespaces",
+    )
+    is_active = models.BooleanField(default=True)
+
+    class Meta:
+        verbose_name = "Namespace"
```

Comment:
verbose names should be lowercase in general

Reply:
Doned